telegram/chatmember.py

```diff
@@ -64,3 +64,11 @@ def de_json(data, bot):
         data['user'] = User.de_json(data.get('user'), bot)
 
         return ChatMember(**data)
+
+    def __eq__(self, other):
+        if isinstance(other, self.__class__):
+            return self.user == other.user and self.status == other.status
+        return NotImplemented
```

Comment:
mmm... following the comments above.
maybe we should use the same `__eq__` from the parent class with a special class like this `_get_id()` will return a Tuple of (user, status)?